src/types/PaymentSheet.ts

```diff
@@ -40,7 +44,20 @@ export declare namespace PaymentSheet {
         merchantCountryCode?: string;
         testEnv?: boolean;
       };
-
+  export interface Address {
```

Comment:
how do users see documentation for these things? I didn't find it in `/docs`, do they use the native SDK docs?

Reply:
That's a good question, this SDK (react native) isn't documented well. There will need to be some effort made to get that fixed. Not sure what that looks like in the short-term, there's many types and interfaces not documented at all....